ui-v2/app/styles/components/list-row/skin.scss

```diff
@@ -21,3 +21,10 @@
 %list-row-detail {
   color: $gray-500;
 }
+%list-row-detail a {
+  color: inherit;
+}
+%list-row-detail a:hover {
+  color: $color-action;
+  text-decoration: underline;
+}
```

Comment:
Any links in the detail portion of a row will now act like the node one (grey hover-to-blue)